fluss-connectors/fluss-connector-spark/fluss-connector-spark-common/src/main/scala/com/alibaba/fluss/connector/spark/SparkTable.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.fluss.connector.spark
+
+import com.alibaba.fluss.config.Configuration
+import com.alibaba.fluss.connector.spark.catalog.SparkCatalog
+import com.alibaba.fluss.connector.spark.utils.SparkTypeUtils
+import com.alibaba.fluss.metadata.{PartitionSpec, TableInfo}
+
+import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.catalyst.expressions.{BoundReference, Cast}
+import org.apache.spark.sql.connector.catalog.{SupportsPartitionManagement, SupportsRead, SupportsWrite, Table, TableCapability, TableCatalog}
+import org.apache.spark.sql.connector.read.ScanBuilder
+import org.apache.spark.sql.connector.write.{LogicalWriteInfo, WriteBuilder}
+import org.apache.spark.sql.types.{StringType, StructType}
+import org.apache.spark.sql.util.CaseInsensitiveStringMap
+
+import java.util
+
+case class SparkTable(catalog: SparkCatalog, flussConfig: Configuration, table: TableInfo)
```

Comment:
Is only `flussConfig: Configuration` and `table: TableInfo` needed here?

Reply:
`case class SparkTable(flussConfig: Configuration, table: TableInfo)`

Reply:
sorry I misunderstood your mean, SparkCatalog is for create and drop partition ,i think thoes methods should be in catalog。